openapi/paths/lambdas/lambdas@users@{address}@parcels@{x}@{y}@permissions.yaml

```diff
@@ -0,0 +1,44 @@
+get:
+  operationId: getParcelPermissions
+  summary: Get the permissions for a parcel
+  tags:
+    - Lambdas
+  description: Get the permissions a user has for a parcel
+  parameters:
+    - in: path
+      name: address
+      required: true
+      schema:
+        type: string
+        example: '0xddf1eec586d8f8f0eb8c5a3bf51fb99379a55684'
+      description: >-
+        Ethereum address from which you want to get the permissions
+    - in: path
+      name: x
+      required: true
+      schema:
+        type: string
+        example: '100'
+      description: >-
+        X coordinate of the parcel
+    - in: path
+      name: y
+      required: true
+      schema:
+        type: string
+        example: '100'
+      description: >-
+        Y coordinate of the parcel
+
+  responses:
+    '200':
+      description: permissions a user has for a parcel
+      content:
+        application/json; charset=utf-8:
+          schema:
+            $ref: ../../components/schemas/lambdas/parcel-permissions.yaml
+          examples:
+            default:
+              $ref: ../../components/examples/lambdas/200-parcel-permissions.json
+    '500':
+      description: 'Failed to fetch permissions for the parcel'
```

Comment:
Can we develop more meaningful error responses like?
* 404 if the parcel is not found
* 401 if the user does not have permissions in that parcel or this would return 200 OK with empty results?

Reply:
please check again